snackager/Dockerfile

```diff
@@ -1,67 +1,67 @@
 ARG node_version
 
-FROM node:${node_version}-alpine as dev
+# --- Scoper - Remove unused workspaces for Snackager
+FROM node:${node_version}-alpine as scoper
+WORKDIR /app
 
-RUN apk --no-cache add git openssh-client
+# Copy repository
+COPY . .
+# Prune repository and remove unused workspaces
+RUN apk --no-cache add jq && \
+  yarn global add turbo@$(jq --raw-output '.devDependencies.turbo' package.json) && \
+  turbo prune --docker --scope=snack-modules-packager
```

Comment:
This is the biggest "magic sauce" in this PR. This does a couple of things:

1. Load **package.json**
2. Get `"turbo": "<version>"` from **package.json**
3. Install this version of `turbo` globally
4. Drop _all workspaces_ that are NOT being used from the monorepo

Especially step 4 reduces the maintenance of this dockerfile. We can split up things, without having to edit this dockerfile, while maintaining smallest possible image.